containerimages/container-disk-images.md

```diff
@@ -0,0 +1,113 @@
+# KubeVirt container-disk images
+The following images stored at `dockerhub.com/kubevirt` and can be used in Kubevirt tests:
+
+- <b>alpine-container-disk-demo</b>  
```

Comment:
You can make it **bold** with `**bold**`

Reply:
Done